pkgs/development/libraries/readline/8.2.nix

```diff
@@ -3,7 +3,7 @@
 , updateAutotoolsGnuConfigScriptsHook
 , ncurses, termcap
 , curses-library ?
-    if stdenv.hostPlatform.isWindows
+    if stdenv.hostPlatform.isMinGW
```

Comment:
What's the exact reason for changing from checking for windows in general to mingw? Would `isWindows || isMinGW` work better?

Reply:
`isWindows` includes Cygwin. Under Cygwin we want to use ncurses, not termcap.